test/integration/linux/inspec/users_spec.rb

```diff
@@ -5,16 +5,11 @@
   describe user('aerickson') do
     it { should exist }
   end
-
```

Comment:
Replace with you or another team member vs delete.

Reply:
Done.